src/inc/til/bitmap.h

```diff
@@ -130,8 +149,27 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
             _sz(sz),
             _rc(sz),
             _bits(sz.area(), fill),
-            _dirty(fill ? sz : til::rectangle{})
+            _dirty(),
+            _runs{}
+        {
+            if (fill)
+            {
+                _dirty = til::rectangle{ sz };

```

Comment:
oh i loved the derpy ternary

Reply:
No prob, it's coming back since you led me to the union realization in another comment.